src/context/CalendarProvider/CalendarProvider.tsx

```diff
@@ -39,7 +39,7 @@ const CalendarProvider = ({
 }: CalendarProviderProps) => {
   const { zoom: configZoom, maxRecordsPerPage } = config;
   const [zoom, setZoom] = useState<ZoomLevel>(configZoom);
-  const recordsThreshold = maxRecordsPerPage ? maxRecordsPerPage : 10;
+  const recordsThreshold = maxRecordsPerPage ? maxRecordsPerPage : 50;
```

Comment:
Move 50 to constants

Reply:
You also can define default values when destructing props
const Comp = ({ maxRecordsPerPage = 50 }: CompProps ) => { ... }